mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -67,81 +70,67 @@ def __clean(self, wdc_jsonl_file_path: Path):
 
         return new_file_name
 
-    def __find_dimensions(self, description, listing, additional_info):
-        """
-        Extract dimension data using regex
-        """
-        dimensions = []
-
-        if description != None:
-            dimensions = re.findall(
-                "\d+(?: \d+)?\s?\w*\sx\s\d+\
-                    (?: \d+)?\s?(?:x\s\d+\s?)?\w*",
-                description,
-            )
-
-        if len(dimensions) == 0:
-            if description:
-                dimensions = re.findall(
-                    "\d+\s?\w+\s\d+\s?\w+\
-                        \slead\sx\s\d+\s?\w+",
-                    description,
-                )
-
-        if len(dimensions) == 0:
-            dimensions = re.findall(
-                "\d+\s?\w*\sx\s\d+\
-                    \s?\w*",
-                listing,
-            )
-
-        if len(dimensions) == 0:
-            dimensions = re.findall(
-                "\d+\s?\w+\s\d+\s?\w+\
-                    \slead\sx\s\d+\s?\w+",
-                listing,
-            )
-
-        if len(dimensions) == 0:
-            if additional_info != None:
-                dimensions = re.findall(
-                    "\d+(?: \d+)?\s?\w*\sx\s\d+\
-                        (?: \d+)?\s?(?:x\s\d+\s?)?\w*",
-                    additional_info,
-                )
-
-            if dimensions:
-                return dimensions
-
-        # dimensions = re.findall("\d+\s?\w+\s\d+\s?\w+\
-        #         \slead\sx\s\d+\s?\w+", additional_info)
-
-        return dimensions
+    # def __find_dimensions(self, description, listing, additional_info):
```

Comment:
Just nuke this code rather than checking in a bunch of commented lines. If you use one commit to remove it with an appropriate message, you can find it later.